consensus/clique/snapshot.go

```diff
@@ -126,11 +126,16 @@ func (s *Snapshot) copy() *Snapshot {
 	return cpy
 }
 
+// checkVote returns if it's a valid vote or not
+func (s *Snapshot) checkVote(address common.Address, authorize bool) bool {
```

Comment:
Thanks the feedback. Update in commit `db6e6950029e97be34b9aa27c5a05ba44495091d`.